### ISSUE TITLE
sysvar(2) could not read, sysvar (1) can

### ISSUE THREAD

**Issue by cyliamargatroid** [Windblade-GR01/Ikemen_GO#73]
In Actual MUGEN, Characters getting hit while lying down uses sysvar(2).
In IKEMEN, Character changes the anim to 0 instead of using sysvar(2) due to it being unrecognized in engine when getting hurt while lying down.
I don't think this has been Implemented yet.

**Comment by Windblade-GR01** [comment]
It's actually used in old versions it seems that the bug was introduced in new ones.
I'll search when the change was made.

**Comment by Windblade-GR01** [comment]
Fixed.
It was a overflow. (Yeah like a old NES game variable overflow bug but with constants)

More specifically a overflow with 3 Constant (The only of 2 things that can overflow in Go)
Me and neatunsou are at fault.
When adding AiLevelF and Const240/640/720p we did not account for that and ended up filling already used space.

Commit with the fix.
9fa534babb5e3bb894f83f80a9a16aa844ba4508
The changes related to the bug are in bytecode.go
